src/vdaf.rs

```diff
@@ -15,6 +15,10 @@ use std::convert::TryFrom;
 use std::fmt::Debug;
 use std::io::Cursor;
 
+/// Domain-separation tag used to bind the VDAF operations to the document version. This will be
+/// revised with each draft with breaking changes.
+pub(crate) const VERSION: &[u8] = b"vdaf-03";
```

Comment:
Would it be useful for the compiler to know the length?
```suggestion
pub(crate) const VERSION: [u8; 7] = b"vdaf-03";
```

Reply:
I don't think so, the codegen should be practically identical either way. We'd just move some ampersands around, and then the reference to an array would be dereferenced into a slice.